pyserini/search/_base.py

```diff
@@ -133,14 +133,14 @@
     'dpr-wq-test': JTopics.DPR_WQ_TEST,
     'dpr-squad-test': JTopics.DPR_SQUAD_TEST,
     'dpr-curated-test': JTopics.DPR_CURATED_TEST,
-    'dpr-trivia-test-gar-t5-answers': JTopics.DPR_TRIVIA_TEST_GART5_ANSWERS,
-    'dpr-trivia-test-gar-t5-titles': JTopics.DPR_TRIVIA_TEST_GART5_TITLES,
-    'dpr-trivia-test-gar-t5-sentences': JTopics.DPR_TRIVIA_TEST_GART5_SENTENCES,
-    'dpr-trivia-test-gar-t5-all': JTopics.DPR_TRIVIA_TEST_GART5_ALL,
-    'nq-test-gar-t5-answers': JTopics.NQ_TEST_GART5_ANSWERS,
-    'nq-test-gar-t5-titles': JTopics.NQ_TEST_GART5_TITLES,
-    'nq-test-gar-t5-sentences': JTopics.NQ_TEST_GART5_SENTENCES,
-    'nq-test-gar-t5-all': JTopics.NQ_TEST_GART5_ALL,
+    # 'dpr-trivia-test-gar-t5-answers': JTopics.DPR_TRIVIA_TEST_GART5_ANSWERS,
```

Comment:
What's the difference between the commented out lines and above lines? clean up?

Reply:
ah... didn't want to make this change. reverted.